src/pages/workspace/reimburse/WorkspaceRateAndUnitPage.js

```diff
@@ -180,7 +180,8 @@ class WorkspaceRateAndUnitPage extends React.Component {
                             }
                         >
                             <TextInput
-                                role={CONST.ACCESSIBILITY_ROLE.TEXT}
+                                accessibilityLabel="Text input field"
```

Comment:
What is this label for? Also, shouldn't this be in `CONST` as well?

Reply:
This should be translated as well.

Reply:
That's outside the scope of this changed this is meant to be a minimal 1-to-1 find/replace of equivalent values.

If I start making other changes then this will require significant testing and greatly increase the risk of introducing new bugs/errors.

Reply:
I don't think this is acceptable. Did we discuss setting this value in the chat?